vault/identity_store_group_aliases.go

```diff
@@ -147,6 +163,12 @@ func (i *IdentityStore) handleGroupAliasUpdateCommon(ctx context.Context, req *l
 		return logical.ErrorResponse("missing alias name"), nil
 	}
 
+	// Get group alias metadata
+	metadata, isMetadataSet, err := d.GetOkErr("metadata")
+	if err != nil {
+		return logical.ErrorResponse(fmt.Sprintf("failed to parse metadata: %v", err)), nil
+	}
+
 	mountAccessor := d.Get("mount_accessor").(string)
```

Comment:
This code and the same code for `name` as they are currently written require both the name and mountAccessor to be set when doing an **update** to a group alias. This is different from doing an update to an entity alias which only requires the field that is changing to be set. I'm not sure if that is intentional. If you want I can look into making updates to group aliases behave similarly to updates entity aliases.